src/classes/BDI_CustObjMappingGAUAllocation.cls

```diff
@@ -73,9 +72,12 @@ public with sharing class BDI_CustObjMappingGAUAllocation extends BDI_ObjectMapp
                     if (objWrap.sObj.Id == null || targetFieldDescribe.isUpdateable()) {
                         // If the target field mapping is one of the key parent objects, then extract the values for later logic
                         if (fieldMapping.Target_Field_API_Name == PAYMENT_FIELDNAME) {
-                            paymentIdValue = value;
+                            // Skip payment lookup since we will not directly link a GAU Allocation to Payment
+                            continue;
```

Comment:
If a current customer has a mapping like this, will they be informed with an error that it no longer works? I guess it will probably reach a point where we don't have payment/object information.

Reply:
Theoretically yes, but it is an extremely unlikely scenario that they would be modifying mappings on the version of the GAU allocation that we ship with the product to map a different value into the payment field. Any changes to mapping for the lookups on GAU Allocation would likely result in failures before or after this change.